db.js

```diff
@@ -424,37 +424,41 @@ var loans = (function () {
         return Promise.reject();
       } else {
         var insertQuery =
-          "INSERT INTO loans book_id, requester_contact, owner_contact, status) " +
-          "VALUES (?,?,?,?)";
-        var insertParams = [
-          loan.book_id,
-          loan.requester_contact,
-          " ",
-          "pending",
-        ];
+          "INSERT INTO loans (book_id, requester_id, owner_contact, status) " +
```

Comment:
Is `requester_contact` still a required field in the database?

Reply:
`owner_contact` and `requester_contact` are still NOT NULL fields, so I would think that you would have to insert a value for both of them?